tests/TestCase/ConfigurationTest.php

```diff
@@ -73,11 +85,19 @@ public function dataTestConfigurationTree()
                         'posts' => [
                             'class' => 'App\Entity\Post',
                             'enable_serializer_groups' => false,
+                            'serializer_groups' => ['searchable'],
                             'index_if' => null,
                         ],
                         'tags' => [
                             'class' => 'App\Entity\Tag',
                             'enable_serializer_groups' => true,
+                            'serializer_groups' => ['searchable'],
+                            'index_if' => null,
```

Comment:
No, this cannot be removed since Symfony Config normalized the structure of the array to the form defined by the semantic configuration (and thus the assertion at `tests/TestCase/ConfigurationTest.php:25` will fail).